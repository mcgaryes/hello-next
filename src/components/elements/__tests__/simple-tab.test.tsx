import {render} from '@testing-library/react'
import SimpleTab from "@/elements/simple-tab";
import {Tab} from "@headlessui/react";

test('Renders simple-tab unchanged', () => {
    const {container} = render(
        <Tab.Group>
            <Tab.List>
                <SimpleTab>Tab</SimpleTab>
            </Tab.List>
        </Tab.Group>
    )
    expect(container).toMatchSnapshot()
})
