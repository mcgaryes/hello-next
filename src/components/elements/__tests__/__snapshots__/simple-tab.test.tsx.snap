// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders simple-tab unchanged 1`] = `
<div>
  <div
    aria-orientation="horizontal"
    role="tablist"
  >
    <button
      aria-selected="true"
      class="bg-blue-500 text-white px-4 py-2"
      data-headlessui-state="selected"
      id="headlessui-tabs-tab-:r0:"
      role="tab"
      tabindex="0"
      type="button"
    >
      Tab
    </button>
  </div>
</div>
`;
